### PR TITLE
Database Refactor

### PR DESCRIPTION
**Pull request by kamilprzyb**
Good luck reviewing that Lol

### CODE REVIEW

File: src/CNLoginServer.cpp [hunk 1]
```diff
@@ -162,7 +170,7 @@ void CNLoginServer::login(CNSocket* sock, CNPacketData* data) {
     DEBUGLOG(
         std::cout << "Login Server: Login success. Welcome " << userLogin << " [" << loginSessions[sock].userID << "]" << std::endl;
     )
-
+
```
Comment (Raymonf) [inline src/CNLoginServer.cpp:173]: bruh

File: src/CNLoginServer.cpp [hunk 2]
```diff
@@ -318,18 +339,27 @@ void CNLoginServer::characterCreate(CNSocket* sock, CNPacketData* data) {
...
+        std::cout << "[WARN] Login Server: invalid CHAR_CREATE packet!" << std::endl;
         return invalidCharacter(sock);
-
-    Database::finishCharacter(character);
-
-    Player player = Database::getPlayer(character->PCStyle.iPC_UID);;
+    }
+    if (!Database::finishCharacter(character, loginSessions[sock].userID))
+    {
+    std::cout << "[WARN] Login Server: Database failed to finish character creation!" << std::endl;
```
Comment (Raymonf) [inline src/CNLoginServer.cpp:349]: bruh

File: src/settings.cpp [hunk 3]
```diff
@@ -72,10 +74,12 @@ void settings::init() {
...
     MOTDSTRING = reader.Get("shard", "motd", MOTDSTRING);
+    DBPATH = reader.Get("shard", "dbpath", DBPATH);
     ACCLEVEL = reader.GetInteger("shard", "accountlevel", ACCLEVEL);
     EVENTMODE = reader.GetInteger("shard", "eventmode", EVENTMODE);
     EVENTCRATECHANCE = reader.GetInteger("shard", "eventcratechance", EVENTCRATECHANCE);
+    DISABLEFIRSTUSEFLAG = reader.GetInteger("shard", "disablefirstuseflag", DISABLEFIRSTUSEFLAG);
     MONITORENABLED = reader.GetBoolean("monitor", "enabled", MONITORENABLED);
     MONITORPORT = reader.GetInteger("monitor", "port", MONITORPORT);
-    MONITORINTERVAL = reader.GetInteger("monitor", "interval", MONITORINTERVAL);
+    MONITORINTERVAL = reader.GetInteger("monitor", "interval", MONITORINTERVAL);
```
Comment (Raymonf) [inline src/settings.cpp:84]: bruh

**Review by Raymonf — CHANGES_REQUESTED** [review]
seeing sqlite_orm.h gone makes me happy :)

**Comment by CakeLancelot** [comment]
Merged manually.

**Pull request closed without merging**